src/jvmTest/kotlin/software/momento/kotlin/sdk/CacheClientControlTest.kt

```diff
@@ -39,4 +42,23 @@ class CacheClientControlTest: BaseJvmTestClass() {
             assert((deleteResponse as CacheDeleteResponse.Error).cause is NotFoundException)
         }
     }
+
+    @Test
+    fun listCacheHappyPath() = runTest  {
+        val cacheName = "kotlin-jvm-create-delete-${UUID.randomUUID()}"
+        var createResponse = cacheClient.createCache(cacheName)
+        assert(createResponse is CacheCreateResponse.Success)
+
+        var listCachesResponse = cacheClient.listCaches()
+        assert(listCachesResponse is CacheListResponse.Success)
+
+        val caches = (listCachesResponse as CacheListResponse.Success).caches
+        assertTrue(caches.size > 1, "There should be exactly 2 caches in the response")
```

Comment:
I think the assertion is correct, but we don't know that there will be only two caches.